common/reactive-grpc-common/src/main/java/com/salesforce/reactivegrpc/common/AbstractStreamObserverAndPublisher.java

```diff
@@ -67,6 +67,7 @@ public void request(long n) {
 
     private final Queue<T> queue;
     private final int prefetch;
+    private final int limit;
```

Comment:
Can you add a comment explaining the difference between `prefetch` and `limit`?

Reply:
added